<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLineEdit" name="interest_rate"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="credit_duration"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Срок кредита (лет)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="credit_sum">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Тип ежемесячных платежей</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Сумма кредита</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>48</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Расчет ежемесячного платежа</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>49</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QLineEdit" name="monthly_payment"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="overpayment"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Общая выплата</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Переплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: lightgray;
    color: black;
    border-radius: 5px;
    padding: 10px;
}</string>
      </property>
      <property name="text">
       <string>Ежемесячный платеж</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="total_payment"/>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="annuity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитетные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="differential">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #3498db, stop:1 #2980b9);
    border: none;
    color: white;
    padding: 10px 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #2980b9, stop:1 #3498db);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #3498db, stop:1 #2980b9);
    border: none;
    color: white;
    padding: 10px 20px;
    border-radius: 5px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #2980b9, stop:1 #3498db);
}</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
